source/trackers.json

```diff
@@ -610,6 +610,10 @@
         "applvn.com": "applovin",
         "applovin.com": "applovin",
         "appsflyer.com": "appsflyer",
+        "bitbucket.org": "atlassian.net",
```

Comment:
@jellizaveta should these be labelled under atlassian, or atlassian.net?

It turned out Atlassian [lists](https://support.atlassian.com/organization-administration/docs/ip-addresses-and-domains-for-atlassian-cloud-products/) all its domains on their support page.

Reply:
should be `atlassian.net` , as this is the name of the tracker